tests/www/test_views.py

```diff
@@ -2535,7 +2531,7 @@ def test_operator_extra_link_multiple_operators(self, get_dag_function):
         AirflowLink2 returns 'https://airflow.apache.org/1.10.5/' link
         GoogleLink returns 'https://www.google.com'
         """
-        get_dag_function.return_value = self.dag
+        mock_get_dagbag.return_value.get_dag.return_value = self.dag
```

Comment:
```suggestion
        self.app.dagbag = mock.MagicMock(**{'get_dag.return_vaalue': self.dag})
```

Reply:
It will be simpler.

Reply:
Yes but it has to be done on class level, otherwise this would create side effects